src/Reports/ElementsInUseReport.php

```diff
@@ -59,11 +62,21 @@ public function columns()
                     return '<span class="element__note">' . _t(__CLASS__ . '.None', 'None') . '</span>';
                 },
             ],
-            'Summary' => [
+            'ElementSummary' => [
```

Comment:
When this was `Summary` it would actually call getSummary() early, then call it again during the try/catch below. This is a minor performance optimisation as well as essential for the bug fix.